docs/pyroomacoustics.rst

```diff
@@ -20,12 +20,14 @@ Submodules
 
    pyroomacoustics.acoustics
    pyroomacoustics.beamforming
+   pyroomacoustics.build_rir
    pyroomacoustics.geometry
    pyroomacoustics.metrics
    pyroomacoustics.multirate
    pyroomacoustics.parameters
    pyroomacoustics.recognition
    pyroomacoustics.room
+   pyroomacoustics.scnr
```

Comment:
Is this a new submodule ?

Reply:
oops I didn't realize that making the docs would create this. and yes I'm about to make a pull request with this soon.

Reply:
should I remove it from this PR?

Reply:
I guess it is better to include it with the relevant code.